### ISSUE TITLE
[ACCESSIBILITY] The screen reader is unable to read the options of the dropdown lists

### ISSUE THREAD

**Issue by clav1d** [rokwire/events-manager#961]
### Description
If the user uses Tab & Shift-Tab to navigate the dropdown lists, the screen reader is unable to read the options

### Acceptance Criteria
The screen reader can read the options of the dropdown lists.
